src/js/ko-components/config/dictionaries.coffee

```diff
@@ -3,16 +3,19 @@
 @OPL.OrificeCalculator.Config ||= {}
 @OPL.OrificeCalculator.Config.Dictionaries ||= {}
 
-@OPL.OrificeCalculator.Config.Dictionaries.PipeID = {
-  twoZeroInch: "2.067'' Sch 40, STD, Sch 40S"
-  oneNineInch: "1.939'' XS, Sch 80, Sch 80S" # Default
-  twoOneInch: "2.157'' Sch 10, Sch 10S"
-  twoTwoInch: "2.245'' Sch 5, Sch 5S"
-  oneSixInch: "1.687'' Sch 160"
-  oneHalfInch: "1.503'' XXS"
-}
-@OPL.OrificeCalculator.Config.Dictionaries.OperatingPressureRead = { gauge: 'Gauge', absolute: 'Absolute' }
-@OPL.OrificeCalculator.Config.Dictionaries.OperatingPressureUnits = {
+@OPL.OrificeCalculator.Config.Dictionaries.PipeID =
+  2.067: "2.067'' Sch 40, STD, Sch 40S"
```

Comment:
You should not use . on variable naming.
. is used as namespacing and has a very specific meaning. please find another way of dealing with this